distribution/test/itests/test-itests-common/src/main/java/org/codice/ddf/itests/common/ServiceManagerImpl.java

```diff
@@ -263,16 +289,36 @@ private FeaturesService getFeaturesService() throws InterruptedException {
                 }
                 Thread.sleep(1000);
             }
+
         }
 
         return featuresService;
     }
 
```

Comment:
For line 265 - 295, is it possible to use `WaitCondition`? It might help give it all a consistent look & feel if it works. 
Could take advantage of the fact the we catch the `NullPointerException` and eliminate `if` null checks.
I'm thinking something like this:
```
         WaitCondition.expect("Waiting service respond")
                .checkEvery(1, TimeUnit.DAYS.SECONDS)
                .within(FEATURES_AND_BUNDLES_TIMEOUT, TimeUnit.MILLISECONDS)
                .until(() -> {
                    try {
                        ServiceReference<FeaturesService> featuresServiceRef =  FrameworkUtil.getBundle(getClass()).getBundleContext().getServiceReference(FeatureService.class);
                        featureService = getBundleContext.getService(featuresServiceRef);
                        return true;
                    } catch (NullPointerException e) {
                        //Unable to get service on this try
                        return false;
                    }
                });

        if(featureService == null) {
            fail(String.format("Feature service could not be resolved within %d minutes.",
                    TimeUnit.MILLISECONDS.toMinutes(FEATURES_AND_BUNDLES_TIMEOUT)));
        }

        return featureService;
```